.gitattributes

```diff
@@ -1,2 +1,5 @@
 * text eol=lf
 *.png binary
+
+# See https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github
+lib/**/*.cjs linguist-generated
```

Comment:
We also have some `.cjs` files in tests, e.g. `lib/__tests__/fixtures/plugin-warn-about-foo.cjs`.

I am assuming that this would also hide those?
Kan we keep them instead?

Reply:
Oh, great catch! I'll find a way to exclude some test files. 👀

Reply:
I've just pushed f48b46f83 to exclude test files. How about it?

Reply:
Looks awesome! Thank you 🙇